Traceback (most recent call last):
  File "/home/jetson/catkin_ws/src/yolo11_pkg/scripts/onnx.py", line 158, in <module>
    detector.run_video_detection()
  File "/home/jetson/catkin_ws/src/yolo11_pkg/scripts/onnx.py", line 138, in run_video_detection
    processed_frame = self.process_frame(frame)
  File "/home/jetson/catkin_ws/src/yolo11_pkg/scripts/onnx.py", line 114, in process_frame
    output = self.session.run(None, {self.input_name: img_data})
  File "/home/jetson/miniconda3/envs/yolo11/lib/python3.10/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 220, in run
    return self._sess.run(output_names, input_feed, run_options)
onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Unexpected input data type. Actual: (tensor(float)) , expected: (tensor(float16))

